modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo/DriverInfoForm.tsx

```diff
@@ -369,68 +289,134 @@ const DriverInfoForm = ({
 
 			<ClayForm>
 				<div className="row">
-					<div className="col filled form-condensed form-group">
+					<div
+						className={classNames(
+							'col filled form-condensed form-group',
+							{
+								'has-error': hasError.firstName,
+							}
+						)}
+					>
 						<ClayInput
 							autoComplete="off"
-							className="bg-neutral-0 firstName w-100"
 							id="firstName"
 							name="firstName"
+							onBlur={(event) => {
+								setHasError({
+									...hasError,
+									firstName:
+										event.target.value === ''
+											? true
+											: false,
+								});
+							}}
 							onChange={(event) => {
 								handleChangeField(
 									'firstName',
 									event.target.value,
 									id
 								);
+								setHasError({
+									...hasError,
+									firstName:
+										event.target.value === ''
+											? true
+											: false,
+								});
 							}}
 							required
 							type="text"
+							value={form[formNumber - 1].firstName}
 						/>
 
 						<label htmlFor="firstName">
 							First Name&nbsp;
 							<span className="text-danger-darken-1">*</span>
 						</label>
+
+						{hasError.firstName && <ErrorMessage />}
 					</div>
 
-					<div className="col filled form-condensed form-group">
+					<div
+						className={classNames(
+							'col filled form-condensed form-group',
+							{
+								'has-error': hasError.lastName,
+							}
+						)}
+					>
 						<ClayInput
 							autoComplete="off"
-							className="bg-neutral-0 lastName w-100"
 							id="lastName"
 							name="lastName"
+							onBlur={(event) => {
+								setHasError({
+									...hasError,
+									lastName:
+										event.target.value === ''
+											? true
+											: false,
+								});
+							}}
 							onChange={(event) => {
 								handleChangeField(
 									'lastName',
 									event.target.value,
 									id
 								);
+								setHasError({
+									...hasError,
+									lastName:
+										event.target.value === ''
+											? true
+											: false,
+								});
 							}}
 							required
 							type="text"
+							value={form[formNumber - 1].lastName}
 						/>
 
-						<label htmlFor="lastName">
+						<label htmlFor="lasstName">
```

Comment:
Please takeout the extra 's'
```suggestion
						<label htmlFor="lastName">
```